modules/tinymce/src/core/main/ts/Rtc.ts

```diff
@@ -372,7 +372,7 @@ export const canApplyFormat = (
 
 export const closestFormat = (
   editor: Editor,
-  names: string): string => getRtcInstanceWithError(editor).formatter.closest(names);
+  names: string[]): string => getRtcInstanceWithError(editor).formatter.closest(names);
```

Comment:
This incorrect type was something we missed when originally reviewing this in https://github.com/tinymce/tinymce/pull/6113/. I noticed it when I added the missing type in `Formatter.ts`.